raiden/tests/fixtures/blockchain.py

```diff
@@ -90,9 +90,10 @@ def pregenerate_dag(request, blockchain_type, dagpath):
     )
 
     if blockchain_type == 'geth' and missing_dag:
-        os.makedirs(os.path.dirname(dagpath))
+        dag_folder = os.path.split(dagpath)[0]
```

Comment:
Here you are taking the directory name two times.

Once with `os.path.split()` and again undernearh with `os.path.dirname()`. Just do it once and perhaps stick to `dirname`.